versions.yaml

```diff
@@ -105,11 +105,6 @@ assets:
           tag: "v5.1.0"
           commit: "d0ed6a69d399ae193959225cdeaa9382746c91cc"
 
-    qemu-experimental:
```

Comment:
So the CI "qemu-experimental" ENV is obsolete now?

Reply:
Well... I'm inclined to think it's pointless complexity, I'm prepared to drop this bit if it comes to it, though.

Reply:
removing experimental :thinking: ... need more eyes here

@jcvenegas @fidencio @wainersm @GabyCT - Do we still need experimental with QEMU 6.1.0 ?

Reply:
@devimc for my tests I don' t need the experimental QEMU.

Reply:
Note that discussion of the merits of this change belong on #2502, I'll be rebasing this patch onto the final version of that one.